en-US/Language.xml

```diff
@@ -527,7 +527,7 @@
 				<term>low-hanging fruit</term>
 				 <listitem>
 					<para>
-						Metaphor. Do not use.
+						Metaphor. Do not use. Instead, refer to what is meant, such as that the results are easily achieved.
```

Comment:
I understand why we're providing an example here, but the suggestion to describe something as "easily achieved" contradicts the guidance to avoid describing something as easy. Consider replacing "easily" in this example.

Additionally, consider either adding a link to the entry on "Avoid stating something is easy", or add text similar to the following: "In general, avoid stating that a task is easy. Focus instead on how to perform the task, or how the outcome benefits the audience."